linux/Platform.c

```diff
@@ -287,8 +287,15 @@ void Platform_setMemoryValues(Meter* this) {
    long int usedMem = pl->usedMem;
    long int buffersMem = pl->buffersMem;
    long int cachedMem = pl->cachedMem;
-   usedMem -= buffersMem + cachedMem;
    this->total = pl->totalMem;
+
+   if (Settings_getSettings()->usedMemOnly) {
```

Comment:
The settings are already part of the value passed in `pl` by looking up `pl->settings`. Please don't introduce any unnecessary globals …

Reply:
Yep, that pointer is exactly what I've been looking for, thanks for hint.